terraform/modules/autoscaler/main.tf

```diff
@@ -27,14 +27,14 @@ resource "google_service_account" "scaler_sa" {
   display_name = "Autoscaler - Scaler Function Service Account"
 }
 
```

Comment:
This change is unrelated to uniform bucket access.
Please revert it.

Reply:
sorry about that, inadvertent change due to some testing I was doing.